### DISCUSSION TITLE
Foscam Floodlight - createSubscription error Error: ONVIF SOAP Fault: HTTP Error: 401 Unauthorized

### DISCUSSION THREAD

**Discussion by jgbaum** [koush/scrypted#1125] [Q&A]
The video on my [Foscam Floodlight](https://www.foscammall.com/products/foscam-security-motion-tracking-floodlight-camera) is working well with the ONVIF plugin and is accessible through HomeKit.  However, I see a bunch of log entries similar to ```createSubscription error Error: ONVIF SOAP Fault: HTTP Error: 401 Unauthorized``` and believe this is why motion events are not being triggered.  Maybe this is related to issues observed with Tapo cameras (e.g., #796)?

Any assistance would be appreciated and I'm happy to dig up more information on my end if it's helpful to the community.

**Comment by MrSco** [comment]
I’m also seeing this with my Foscam r2 camera. Did you ever figure it out?

**Reply by jgbaum** [reply]
For now, I've given up on the ONVIF plugin and am using the RTSP plugin instead, since it seems to better support HKSV recording.  I just posted a related topic [here](https://github.com/koush/scrypted/discussions/1143).

I'm using the RTSP stream from go2rtc, which mostly worked out of the box.  The only setting that I needed to change was to set the 'RTSP Parser' to 'Scrypted (TCP)' on the stream.

**Comment by dohlin** [comment]
I'm seeing this error too, but the main "issue" I'm seeing is that the Onvif PTZ function simply does not work with the R2...which is a definite bummer.  I know it isn't the best camera, but it was cheap and for an indoor, local-only camera it does an ok job.
